projects/collection-editor-library/src/lib/services/editor/editor.service.ts

```diff
@@ -260,5 +263,61 @@ export class EditorService {
     this.telemetryService.error(telemetryErrorData);
   }
 
+  getAssetMedia(req?: object) {
+    const reqParam = {
+      url: 'composite/v3/search',
+      data: {
+        request: {
+          filters: {
+            contentType: 'Asset',
+            compatibilityLevel: {
+              min: 1,
+              max: 2
+            },
+            status: ['Live'],
+          },
+          limit: 50,
+        }
+      }
+    };
+    reqParam.data.request = req ? _.merge({}, reqParam.data.request, req) : reqParam;
+    return this.publicDataService.post(reqParam);
+  }
+
+  createMediaAsset(req?: object) {
+    const reqParam = {
+      url: 'content/v3/create',
+      data: {
+        request: {
+          content: {
+            contentType: 'Asset',
+            language: ['English'],
+            code: UUID.UUID(),
+          }
+        }
+      }
+    };
+    reqParam.data.request = req ? _.merge({}, reqParam.data.request, req) : reqParam;
+    return this.publicDataService.post(reqParam);
+  }
+
+  uploadMedia(req, assetId: any) {
+    let reqParam = {
+      url: `content/v3/upload/${assetId}`,
+      data: req.data
+    };
+    reqParam = req ? _.merge({}, reqParam, req) : reqParam;
+    return this.publicDataService.post(reqParam);
+  }
+
+  generatePreSignedUrl(req, contentId: any) {
+    const reqParam = {
+      url: `content/v3/upload/url/${contentId}`,
+      data: {
+        request: req
+      }
+    };
+    return this.publicDataService.post(reqParam);
+  }
```

Comment:
@rajnish-dargan We have already created this `getAssetMedia` method in the `question.service.ts` file. 
Please check the same  and use it from here : `question.service.ts`

cc: @pallakartheekreddy

Reply:
@pallakartheekreddy Updated the changes please review once.